crates/bevy_render/src/lib.rs

```diff
@@ -346,27 +303,33 @@ struct ScratchMainWorld(World);
 /// Executes the [`Extract`](RenderStage::Extract) stage of the renderer.
 /// This updates the render world with the extracted ECS data of the current frame.
 fn extract(app_world: &mut World, render_app: &mut App) {
-    let extract = render_app
-        .schedule
-        .get_stage_mut::<SystemStage>(RenderStage::Extract)
-        .unwrap();
+    let mut extract = render_app.world.remove_resource::<ExtractStage>().unwrap();
```

Comment:
Could this be done with `World::resource_scope`?

Reply:
done